test/write.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+let should = require('should'),
+    express = require('express'),
+    request = require('supertest'),
+    mung = require('../');
+
+describe('mung write', () => {
+
+    let originalResponseTextBody
+    let originalResponseJSONBody
+    let modifiedResponseTextBody
+
+    beforeEach(() => {
+        originalResponseTextBody = 'This is the response body'
+        modifiedResponseTextBody = 'This is the response body with more content';
+        originalResponseJSONBody = {
+            a: 'a'
+        }
+    })
+
+    function appendText (chunk, req, res) {
+        return (chunk + ' with more content')
```

Comment:
I know you mentioned that `mung.json` has the ability to modify the passed-in object in place, but in the case of res.write, since we are dealing with immutable strings and/or buffers, the function will have to return the manipulated value. I can add this to the README along with the description of this function.